docs/admin/authorization/rbac.md

```diff
@@ -517,31 +597,35 @@ To bootstrap initial roles and role bindings:
 
 ## Command-line utilities
 
-Two `kubectl` commands exist to grant roles to users, within a namespace, or across the entire cluster.
+Two `kubectl` commands exist to grant roles within a namespace or across the entire cluster.
 
 ### `kubectl create rolebinding`
 
-Grants a role or clusterrole within a specific namespace. Examples:
+Grants a `Role` or `ClusterRole` within a specific namespace. Examples:
 
 * Grant the `admin` `ClusterRole` to a user named `bob` in the namespace `acme`:
 
-    `kubectl create rolebinding my-role-binding --clusterrole=admin --user=bob --namespace=acme`
+    `kubectl create rolebinding bob-admin-binding --clusterrole=admin --user=bob --namespace=acme`
 
 * Grant the `view` `ClusterRole` to a service account named `myapp` in the namespace `acme`:
 
-    `kubectl create rolebinding my-role-binding --clusterrole=view --serviceaccount=acme:myapp --namespace=acme`
+    `kubectl create rolebinding myapp-view-binding --clusterrole=view --serviceaccount=acme:myapp --namespace=acme`
 
 ### `kubectl create clusterrolebinding`
 
 Grants a `ClusterRole` across the entire cluster, including all namespaces. Examples:
 
-* Grant the `cluster-admin` `ClusterRole` to a user named `root`:
+* Grant the `cluster-admin` `ClusterRole` to a user named `root` across the entire cluster:
+
+    `kubectl create clusterrolebinding root-cluster-admin-binding --clusterrole=cluster-admin --user=root`
+
+* Grant the `system:node` `ClusterRole` to a user named `kubelet` across the entire cluster:
 
-    `kubectl create clusterrolebinding my-root-binding --clusterrole=cluster-admin --user=root`
+    `kubectl create clusterrolebinding kubelet-node-binding --clusterrole=system:node --user=kubelet`
 
-* Grant the `system:node` `ClusterRole` to a user named `kubelet`:
+* Grant the `view` `ClusterRole` to a service account named `myapp` in the namespace `acme` across the entire cluster:
```

Comment:
- [ ] Likewise, here I'd write "myapp" and "acme" as quoted strings, since they're not reserved words or tokens that we're documenting here.
> named "myapp" in the namespace "acme" across